src/Input.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+import { placeholder } from 'polished';
+
+import {
+  borderRadius,
+  borderWidth,
+  colors,
+  fontSizes,
+  lineHeights,
+  spacings
+} from './theme';
+
+import Label from './Label';
+import Paragraph from './Paragraph';
+
+const StyledInput = styled.input`
+  border-radius: ${borderRadius.small};
+  border: ${borderWidth.small} solid ${colors.grey20};
+  box-sizing: border-box;
+  color: ${colors.black};
+  font-size: ${fontSizes.size2};
+  line-height: ${lineHeights.small};
+  padding: ${spacings.smaller};
+  width: 100%;
+
+  ${placeholder({ color: colors.grey50 })};
+
+  &:focus {
+    border: ${borderWidth.small} solid ${colors.blueLight};
+    box-shadow: 0px 0px 8px rgba(68, 85, 255, 0.1);
```

Comment:
Are we allowed shorthand here? 0 0 8px v.s. 0px 0px 8px?

Reply:
Good call. Also going to look into some linting for something like this.

Reply:
That `rgba` should probably be using one of our colors as well.